conformance/utils/kubernetes/helpers.go

```diff
@@ -993,3 +994,36 @@ func findPodConditionInList(t *testing.T, conditions []v1.PodCondition, condName
 	tlog.Logf(t, "%s was not in conditions list", condName)
 	return false
 }
+
+// BackendTLSPolicyMustHaveCondition checks that the created BackentTLSPolicy has the Condition,
+// halting after the specified timeout is exceeded.
+func BackendTLSPolicyMustHaveCondition(t *testing.T, client client.Client, timeoutConfig config.TimeoutConfig, policyNN, gwNN types.NamespacedName, condition metav1.Condition) {
+	t.Helper()
+	// Use HTTP Route timeout condition. This way we are sure that route for a given Service was applied.
```

Comment:
nit:
I don't think this comment is meaningful (or true?).

Reply:
The exact processing time for BTP is implementation-specific, but it should definitely occur before the Route is configured. I added this comment to address potential questions regarding this value, and I can remove it if it's unnecessary.

Reply:
```suggestion
```